gallery.css

```diff
@@ -130,20 +144,21 @@
         min-height: 100vh;
         width: calc(var(--items-mobile) * 100%);
     }
-    .gallery-element {
-        min-height: 100vh;
+    .gallery-controls {
+        width: calc(100% / var(--items-mobile));
```

Comment:
CSS variables are not supported very well yet, except for the most modern browsers (released no earlier than one year ago). See http://caniuse.com/#feat=css-variables. This leads to problems on older mobile phones.
You should replace these with fixed values, as you did almost everywhere else. You can also convert this to LESS/SCSS if you want to, too, to be able to use variables and other neat features like color calculation.